drivers/mbed_lib.json

```diff
@@ -41,6 +41,10 @@
         "qspi_csn": {
             "help": "QSPI chip select pin",
             "value": "QSPI_FLASH1_CSN"
+        },
+        "default-adc-vref": {
+            "help": "Default reference voltage for ADC (float)",
+            "value": 3.3
```

Comment:
shouldn't it be a 3.3f ? Otherwise a double to float conversion is performed.

Reply:
I had that but I was seeing some JSON parsing issues. I wasn't sure if the build tools would parse "3.3f" correctly but they do :+1: 

Reintroduced float "f" specifier.